blocks/toc/toc.js

```diff
@@ -0,0 +1,44 @@
+function createEntry(element) {
```

Comment:
@larsauffarth : Please see if scripts/dom-builder.js might help you here, see social.js as an example